examples/pennylane/0_Getting_started/0_Getting_started.ipynb

```diff
@@ -262,7 +262,9 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # device_arn = Devices.Rigetti.AspenM3
</pre><p>Should we be removing this as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>